hpedockerplugin/cmd/cmd_deleteshare.py

```diff
@@ -0,0 +1,104 @@
+import json
+import six
+from oslo_log import log as logging
+
+from hpedockerplugin.cmd import cmd
+from hpedockerplugin import exception
+
+LOG = logging.getLogger(__name__)
+
+
+class DeleteShareCmd(cmd.Cmd):
+    def __init__(self, file_mgr, share_info):
+        self._file_mgr = file_mgr
+        self._etcd = file_mgr.get_etcd()
+        self._fp_etcd = file_mgr.get_file_etcd()
+        self._mediator = file_mgr.get_mediator()
+        self._backend = file_mgr.get_backend()
+        self._share_info = share_info
+        self._cpg_name = share_info['cpg']
+        self._fpg_name = share_info['fpg']
+
+    def execute(self):
+        with self._fp_etcd.get_fpg_lock(self._backend, self._fpg_name):
+            self._delete_share()
```

Comment:
`LOG.info` is needed before delete share is attempted.

Reply:
It's there in self._delete_share() function before invoking delete on mediator